src/handlers/pull_requests_assignment_update.rs

```diff
@@ -17,70 +13,23 @@ impl Job for PullRequestAssignmentUpdate {
     }
 
     async fn run(&self, ctx: &super::Context, _metadata: &serde_json::Value) -> anyhow::Result<()> {
-        let db = ctx.db.get().await;
         let gh = &ctx.github;
 
         tracing::trace!("starting pull_request_assignment_update");
 
-        let rust_repo = gh.repository("rust-lang/rust").await?;
-        let prs = retrieve_pull_requests(&rust_repo, &gh).await?;
-
-        // delete all PR assignments before populating
-        init_table(&db).await?;
-
-        // aggregate by user first
-        let aggregated = prs.into_iter().fold(HashMap::new(), |mut acc, (user, pr)| {
-            let (_, prs) = acc.entry(user.id).or_insert_with(|| (user, Vec::new()));
-            prs.push(pr);
-            acc
-        });
+        let rust_repo = gh.repository("kobzol/bors-kindergarten").await?;
```

Comment:
This definitely isn't right...

Reply:
Oops :) I accidentally committed this during the rebase, good catch!